internal/pfcpsim/server.go

```diff
@@ -237,6 +232,51 @@ func (P pfcpSimService) CreateSession(ctx context.Context, request *pb.CreateSes
 				Build(),
 		}
 
```

Comment:
Should we remove this part and only depend on the part where we generate starting from the appFilters? I'm not sure if this can create any issues about retro-compatibility.

Reply:
This means that we have to specify at least one App filter when issuing a create session command

Reply:
We can have a default behavior (i.e., allow all).